README.md

```diff
@@ -1 +1,40 @@
-# git
\ No newline at end of file
+# git
+
+## Intro
+* Git : A tool track changes in your code 
+* Github : Website that hosts the Git repositories 
+___
+## git terms 
+* clone : make local copy of the remote repo
```

Comment:
Create a table here, with items in one column and description in other.

Reply:
The suggested change have been made